app/services/discovery_report.rb

```diff
@@ -70,26 +66,19 @@ def registration_check(druid)
   # @return [String] primitive version
   def header
     fields = ['Object Container', 'Number of Items', 'Files with no ext', 'Files with 0 Size', 'Total Size', 'Files Readable']
-    fields.concat ['Label', 'Source ID'] if using_manifest?
+    fields.concat ['Label', 'Source ID']
     fields.concat ['Num Files in CM Manifest', 'All CM files found'] if using_smpl_manifest?
-    fields.concat ['Duplicate Filenames?', 'DRUID', 'Registered?', 'APO exists?']
-    fields << 'SourceID unique in DOR?' if using_manifest?
+    fields.concat ['Duplicate Filenames?', 'DRUID', 'Registered?', 'APO exists?', 'SourceID unique in DOR?']
     fields.join(' , ')
   end
 
   # For use by template
   def skipped_files
     files = ['Thumbs.db', '.DS_Store'] # if these files are in the bundle directory but not in the manifest, they will be ignorned and not reported as missing
     files << File.basename(content_md_creation[:smpl_manifest]) if using_smpl_manifest?
-    files << File.basename(manifest) if using_manifest?
```

Comment:
I thought we were requiring a manifest file in all cases -- so the if clause can go, but the line should stay?

Reply:
^^ whoops - I guess the above is a "request changes"

Reply:
Thanks, incorporated 2 lines earlier in the initial array.